### PR TITLE
Home and About content updates

### PR DESCRIPTION
**Pull request by thibaudcolas**
- Updates the Home content to more accurately reflect this year’s plans
- Updates the About page content along the same lines
- Changes the main contact email to be the correct 2023 address (matching what 2022 DjangoCon Europe did)
- Adds the conference’s code of conduct (copy of the 2022 CoC with minor layout changes to leverage the site’s "callout" block)

---

One thing I haven’t done is add the code of conduct response guide. I’ve kept the links to it in the CoC contents, but currently they 404.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -107,9 +107,9 @@ visa_email: "visas@djangocon.eu"
 # Links
 cfp_application: "https://pretalx.com/djangocon-europe-2023/cfp"
 hotel_reservation_link: "https://www.marriott.com/events/start.mi?id=1642551494496&key=GRP"
-# slack_link: "https://djangoconus2023.herokuapp.com"
 opportunity_grant_application: "https://docs.google.com/forms/d/1yc1HqqcJwi669vxTTs1daJbTeiNYwqM2h3gIgqbpWVE/viewform"
 slack_link: ""
+# slack_link: "https://join.slack.com/t/djangoconeurope/shared_invite/zt-1gjg5lqkz-qVQkNnhjztXVme7TQ7ziQA"
```
Comment (thibaudcolas) [inline _config.yml:112]: This is the 2022 DjangoCon Europe link. I’ve copied it here so we have an easier time figuring out where to update it once needed.

File: _includes/schedule-date-cards.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<div class="row column xlarge-9">
```
Comment (thibaudcolas) [inline _includes/schedule-date-cards.html:1]: Extracted to a separate template so we don’t have to copy-paste this between the different pages that display those dates.

File: _pages/conduct.html [hunk 3]
```diff
@@ -21,25 +22,19 @@ <h1>{{ page.heading }}</h1>
   <section class="section-pad">
     <div class="row column">
         <h2 class="demp">
-        DjangoCon US is dedicated to providing a fun harassment-free conference experience for everyone, regardless of gender, gender identity, sexual orientation, disability, physical appearance, body size, race, or religion.
+          We ask each attendant to keep the community in mind and do their best to foster a positive environment for everyone.
```
Comment (thibaudcolas) [inline _pages/conduct.html:25]: All content here is a copy of https://2022.djangocon.eu/conduct/code_of_conduct/.

**Review by knyghty — APPROVED** [review]
Ok looks good enough to me

**Pull request merged**